go.mod

```diff
@@ -5,11 +5,12 @@ go 1.17
 require (
 	github.com/pkg/errors v0.9.1
 	github.com/sirupsen/logrus v1.8.1
+	github.com/spf13/cobra v1.5.0
 	github.com/spf13/pflag v1.0.5
-	google.golang.org/grpc v1.40.0
+	google.golang.org/grpc v1.47.0
 	google.golang.org/protobuf v1.27.1
-	k8s.io/apimachinery v0.24.1
-	k8s.io/client-go v0.24.1
+	k8s.io/apimachinery v0.24.2
```

Comment:
Please revert back with the older version of dependencies

Reply:
done